### DISCUSSION TITLE
Add editor bindings for drop and paste event

### DISCUSSION THREAD

**Discussion by svenadlung** [ueberdosis/tiptap#5613] [Feature Requests]
### Description

It would be great to have [editor bindings](https://tiptap.dev/docs/editor/api/events#option-2-binding) for drop and paste events similar to other events: https://github.com/ueberdosis/tiptap/blob/9e18d243e048ad9c09e5da1cb65eeddff141cc0e/packages/core/src/Editor.ts#L108-L114

### Use Case

It would be consistent ☺️

### Type

Other

**Comment by litingyes** [comment]
I think that **editor bindings** are some events about editor instance itself,  and you can create `ProseMirrorPlugin` to handle other dom events, such as bind paste event: https://github.com/vueditor/rich-text-editor/blob/cca2e60fcd88e8b5c5aaca6a3cb07f0f8bcf7069/src/editor/extensions/image/index.ts#L40

**Comment by nperez0111** [comment]
Resolved with https://github.com/ueberdosis/tiptap/pull/5684
